source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,286 @@
+.. _csharp-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query to match a subset
+of documents using the {+driver-long+}.
+
+Most CRUD operations allow you to narrow the set of matched documents
+by passing an expression as an argument to the method, or by specifying
+matching criteria in a **query filter**. Query filters contain one or more query
+operators to determine which documents to include in the resulting set.
+
+.. tip:: Find All Documents
+
+   Passing an empty query filter matches all documents in the collection.
+
+.. tip:: Using LINQ
+
+   You can also specify queries using LINQ. To learn more about using LINQ, see
+   :ref:`csharp-linq`.
+
+The examples in this guide use the following documents in a collection called
+``guitars``.
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "models": ["Stratocaster", "Telecaster"], "establishedYear": 1946, "rating": 9 }
```

Comment:
Suggestion: The length of these documents makes it so that all output blocks require horizontal scroll to view. Would it be possible to make it such that these documents are less long? Maybe something like

```
{"_id:1, "make":"Fender", "colors":["red","blue"], "qty": 10}
```

Reply:
Changing the model to be shorter will make it so some of the examples don't work. For example, the rating field being added on some of the documents allows demonstration of the Exists() method. I could change "models" to "colors" so that it uses shorter words, but I'm not sure the value of having less horizontal scroll is worth rewriting/retesting every example on the page